pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -56,7 +56,10 @@ const (
 	DikastesContainerName            = "dikastes"
 	ModSecurityRulesetVolumeName     = "modsecurity-ruleset"
 	ModSecurityRulesetVolumePath     = "/etc/modsecurity-ruleset"
+	DefaultCoreRulesetVolumeName     = "coreruleset-default"
+	DefaultCoreRulesetVolumePath     = "/etc/coreruleset-default"
```

Comment:
Please change to `DefaultCoreRulesetVolumePath = "/etc/modsecurity-ruleset/coreruleset-default"` (adapt if you rename `modsecurity-ruleset`)